docker/psi4/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM conda/miniconda3:latest
+
+RUN apt-get update && \
+  apt-get install -y gcc && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN conda update --all && \
+  conda install psi4=1.2 psi4-rt=1.2 python=3.6 -c psi4
```

Comment:
Do we want to fix the version to 1.2 or let conda pick the latest?

Reply:
I would like to be conservative and pin to a version we know works for now.

Reply:
As a rule I think we always want to pin to known versions, and this is important for reproducibility in general. We can move deliberately, and perform testing, but want to avoid upgrades as a side effect of building an image.